code/modules/ranch/ranch_objs.dm

```diff
@@ -128,6 +131,7 @@ TYPEINFO(/obj/submachine/chicken_incubator)
 			incubate_count = 0
 
 		else
+			boutput(user, SPAN_ALERT("<b>This egg is infertile and cannot be used!</b>"))
```

Comment:
This will cause the text "This egg is infertile and cannot be used!" if you hit it with any abritrary object and is unnecessesary